src/test/videopress/__snapshots__/edit.js.snap

```diff
@@ -1,39 +1,39 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Update VideoPress block's settings should update Playback Bar Color section's Dynamic color setting 1`] = `"<!-- wp:videopress/video {"title":"default-title-is-file-name","description":"","id":1,"guid":"AbCdEfGh","privacySetting":2,"allowDownload":false,"rating":"G","isPrivate":true,"duration":2803} /-->"`;
+exports[`Update VideoPress block's settings should update Playback Bar Color section's Dynamic color setting 1`] = `"<!-- wp:videopress/video {"title":"default-title-is-file-name","description":"","id":34,"guid":"AbCdEfGh","privacySetting":2,"allowDownload":false,"rating":"G","isPrivate":true,"duration":1200} /-->"`;
```

Comment:
The `id` and `duration` attributes have been modified due to using the default values specified in the util function (reference). They don't have an impact on the test cases, so we can omit them.

https://github.com/wordpress-mobile/gutenberg-mobile/blob/68a1ccd7f4ab9334b596e042606887bed4162813/src/test/videopress/local-helpers/utils.js#L203-L229